qa/tasks/cephfs/test_exports.py

```diff
@@ -114,11 +117,19 @@ def test_export_pin(self):
             self._wait_subtrees(status, 0, [('/1', 0), ('/1/4/5', 1), ('/a', 1), ('/a/b/aa/bb', 0)])
 
         # Test getfattr
-        self.assertTrue(self.mount_a.getfattr("1", "ceph.dir.pin") == "0")
-        self.assertTrue(self.mount_a.getfattr("1/4", "ceph.dir.pin") == "-1")
-        self.assertTrue(self.mount_a.getfattr("1/4/5", "ceph.dir.pin") == "1")
-        if (len(self.fs.get_active_names()) > 2):
-            self.assertTrue(self.mount_a.getfattr("1/2/3", "ceph.dir.pin") == "2")
+        def test_getfattr():
```

Comment:
Let's also move `test_getfattr` to an actual test (i.e. not a lambda you're calling in `test_export_pin`). `test_export_pin` should no longer call `getxattr` at all.